snarkos/main.rs

```diff
@@ -159,7 +157,7 @@ async fn start_server(config: Config) -> anyhow::Result<()> {
 
     // Enable the sync layer.
     {
-        let memory_pool = MemoryPool::from_storage(&storage).await?;
+        let memory_pool = MemoryPool::new(); // from_storage(&storage).await?;
```

Comment:
```suggestion
        let memory_pool = MemoryPool::new();
```
I think it's reasonable to start with a blank memory pool here? Unless there were any txs we might not have propagated to other nodes yet? Or should the users re-create them?